src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.targets

```diff
@@ -35,6 +35,7 @@ The .NET Foundation licenses this file to you under the MIT license.
     <IlcTreatWarningsAsErrors Condition="'$(IlcTreatWarningsAsErrors)' == ''">$(TreatWarningsAsErrors)</IlcTreatWarningsAsErrors>
     <_IsiOSLikePlatform Condition="'$(_targetOS)' == 'maccatalyst' or $(_targetOS.StartsWith('ios')) or $(_targetOS.StartsWith('tvos'))">true</_IsiOSLikePlatform>
     <_IsApplePlatform Condition="'$(_targetOS)' == 'osx' or '$(_IsiOSLikePlatform)' == 'true'">true</_IsApplePlatform>
+    <WasmExternalDwarf Condition="'$(WasmExternalDwarf)' == ''and '$(_targetOS)' == 'browser' and '$(DotNetJsApi)' != 'true'">true</WasmExternalDwarf>
```

Comment:
There is already a documented property of this in the NAOT toolchain: `StripSymbols`. We can use it.

Also, why disable for `DotNetJsApi`? I think it makes sense for `DotNetJsApi` as well.

Reply:
Probably, I didn't really know, and just copied what was in the publish file for the `.wasm` file.  Will remove it.